### PR TITLE
[DS-2372] - Maintenance google sheets access token could not be refreshed

### PR DESCRIPTION
**Pull request by ivankahitin**
Jira: https://panoply.atlassian.net/browse/DS-2372

Created separate exception for token validation error

### CODE REVIEW

File: panoply/datasource.py [hunk 1]
```diff
@@ -140,7 +140,7 @@ def wrapper(*args, **kwargs):
                              'The user would have to re-authenticate',
                              traceback.format_exc())
                     # raise a non-retryable exception
-                    raise PanoplyException(
+                    raise OathValidationException(
                         'access token could not be refreshed ({})'
                         .format(str(e)), retryable=False)
```
Comment (solomiya-hatalyak) [inline panoply/datasource.py:145]: If you add original error, just call it like that 
```TokenValidationException (e, 'access token could not be refreshed ({})'.format(str(e)), retryable=False)```

**Review by solomiya-hatalyak — CHANGES_REQUESTED** [review]

File: panoply/datasource.py (hunk 1 above)
Comment (ivankahitin) [reply panoply/datasource.py:145]: But in this case we will show original exception twice. First when we put as first parameter and then when when format string

Comment (solomiya-hatalyak) [reply panoply/datasource.py:145]: We will only store the original error as property. And will be able to reach it in our DS.

File: panoply/errors/exceptions.py [hunk 2]
```diff
@@ -25,3 +25,9 @@ def __init__(self, message, code, exception_cls,
         self.database_id = database_id
         self.exception_cls = exception_cls
         self.created_at = datetime.utcnow()
+
+
+class TokenValidationException(PanoplyException):
```
Comment (solomiya-hatalyak) [inline panoply/errors/exceptions.py:30]: Please, import `TokenValidationException` in `panoply-python-sdk/panoply/errors/__init__.py`

**Review by solomiya-hatalyak — APPROVED** [review]
Please, import TokenValidationException in panoply-python-sdk/panoply/errors/__init__.py

**Pull request merged**